docs/dev/api-reference/index.rst

```diff
@@ -1,77 +0,0 @@
-API reference
-=============
-
-.. note::
-
-  API documentation is being moved to PyPI's
-  `user documentation site <https://docs.pypi.org/api/>`_.
-
-  Please see `issue #16541 <https://github.com/pypi/warehouse/issues/16541>`_
-  for more information and status updates.
```

Comment:
I'm thinking we should keep this file for now and add a short note that the docs have moved. (This also gives usa. better place to put the XML-RPC docs rather than at top level).

Reply:
Sounds good, I'll bring this page back.

Reply:
(Just the page, not all the content, just to be clear)

Reply:
Done! I've put the index back in as a stub documenting the migration of everything except XML-RPC.